### PR TITLE
docs(windows): Update OS and whatsnew in help

### PR DESCRIPTION
**Pull request by darcywong00**
Updates in-app help to include OS support for Windows 11.
Also updates whatsnew to reflect 15.0 changes according to the wiki on [new 15.0 functionality](https://github.com/keymanapp/keyman/wiki/Keyman-15.0-New-Functionality).

@keymanapp-test-bot skip

### CODE REVIEW

File: windows/src/desktop/help/common/os.md [hunk 1]
```diff
@@ -15,6 +15,8 @@ Windows operating systems:
 
 -   Windows 10
 
+-   Windows 11
+
```
Comment (rc-swag) [inline windows/src/desktop/help/common/os.md:19]: Are we officially supporting Windows 11 in version 15?

Comment (darcywong00) [reply windows/src/desktop/help/common/os.md:19]: I thought we're officially supporting anything past Windows 7. You or @MakaraSok were able to run Keyman in a Windows 11 virtual box, right?

Comment (rc-swag) [reply windows/src/desktop/help/common/os.md:19]: Yes we do. I just thought early on @mcdurdin had said we would wait till version 16. Mind you that was when 15 was going to be released in March. We have had more time with some Beta testers on Win 11 now.
I just checked the latest acceptance test Makara was working on and he tested Win 11. So I am fine if @mcdurdin  is ok with that.

Comment (darcywong00) [reply windows/src/desktop/help/common/os.md:19]: From discussion in standup, we can have Keyman 15 mention support for Win 11.

**Review by rc-swag — APPROVED** [review]
lgtm

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 15.0.242-beta](https://keyman.com/downloads/releases/beta/15.0.242)

**Pull request merged**